# $Id$

This file contains a patch for lesstif-0.85.  We suggest that this
patch be applied (using `patch -l < lesstif-0.85.patch') before
building LessTif and running DDD with LessTif.

*** lesstif-0.85/lib/Xm/ScrolledW.c.orig	Wed May 27 20:52:23 1998
--- lesstif-0.85/lib/Xm/ScrolledW.c	Fri Jun  5 19:18:02 1998
***************
*** 53,63 ****
  /* rws 25 Mar 1998
     With this undefined the page list of mgv and mfm work correctly. The mfm
     problem can be seen if you start it and then select a directory from the
     middle list.  At this point the list on the right gets real tiny.
   */
! #undef MFM_BUGS
  
  /* Forward Declarations */
  
  #if 1
  static void class_initialize();
--- 53,63 ----
  /* rws 25 Mar 1998
     With this undefined the page list of mgv and mfm work correctly. The mfm
     problem can be seen if you start it and then select a directory from the
     middle list.  At this point the list on the right gets real tiny.
   */
! #define MFM_BUGS 1
  
  /* Forward Declarations */
  
  #if 1
  static void class_initialize();

